maze-consulting/SFCC_Klaviyo_Maze
# Task/kscu 74 clean up

## Description

This is a code clean-up branch that includes a few revisions to account for price ranges in SFRA, bonus products in the added to cart event, and adds logging where comments previously suggested. 

Otherwise, this update consists of syntax adjustments, styling adjustments for consistency, file structure adjustment where needed (ex: recreateCarthelpers placed in the script/klaviyo directory, etc.). 

## Manual Testing Steps
A full suite of tests were performed following these updates. Changes include running through each of the events (ex: _'Searched Site'_, _'Viewed Product'_, _'Viewed Category'_, _'Added to Cart'_, _'Started Checkout'_ and _'Order Confirmation'_). 

Various products were also tested along the way, including: 

- standard products w/o promotions or discounts
- standard products w/ promotions
- standard products w/ discounts
- standard products w/ discounts & promotions
- products with production options
- product bundles w & w/o product options
- product sets 

The klaviyo dashboard was reviewed following each of these events to check for any errors or discrepancies. All situations & events tested as expected.

## Pre-Submission Checklist:
[ x ] This update successfully builds
[ x ] No console errors are displayed on the screen with these updates
[ x ] Console Logs & debugging created during development for debugging that are no longer needed have been removed
[ x ] A description of this update & references to the manual tests has been included in this PR
